proto/agent.proto

```diff
@@ -222,4 +226,41 @@ message AppLaunchConfig {
   string launchUIDescription = 6;
 }
 
+message Onboarding {
```

Comment:
change to ClusterGitoptsConfig

string gitType = 1;
  string projectUrl = 2;
string accessToken
string onboardingStatus